client/src/modules/reports/generate/account_report_multiple/account_report_multiple.config.js

```diff
@@ -0,0 +1,107 @@
+angular.module('bhima.controllers')
+  .controller('account_report_multipleController', AccountReportMultipleConfigController);
+
+AccountReportMultipleConfigController.$inject = [
+  '$sce', 'NotifyService', 'BaseReportService', 'AppCache', 'reportData',
+  '$state', 'moment', 'SessionService',
+];
+
+function AccountReportMultipleConfigController(
+  $sce, Notify, SavedReports, AppCache, reportData, $state,
+  Moment, Session
+) {
+  const vm = this;
+  const cache = new AppCache('configure_account_report_multiple');
+  const reportUrl = 'reports/finance/account_report_multiple';
+  vm.previewGenerated = false;
+  vm.accountIds = [];
+  delete cache.reportDetails;
```

Comment:
Why do we need to delete the `cache.reportDetails`?  Doesn't that defeat the purpose of a cache?